internal/service/privatelinkendpointservice/resource_privatelink_endpoint_service_test.go

```diff
@@ -147,3 +179,26 @@ func configCompleteAWS(awsAccessKey, awsSecretKey, projectID, providerName, regi
 		}
 	`, awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID, resourceSuffix)
 }
+
+func configFailAWS(awsAccessKey, awsSecretKey, projectID, providerName, region, resourceSuffix string) string {
+	return fmt.Sprintf(`
+		provider "aws" {
+			region        = "%[5]s"
+			access_key = "%[1]s"
+			secret_key = "%[2]s"
+		}
```

Comment:
Is the aws provider necessary for this test? I believe at the end we only rely on atlas resources for this failure scenario.

Reply:
You're absolutely right, no need for AWS provider! Removed 👍